### ISSUE TITLE
ACC-270.6(Level A)_Kyma-dashboard_Combo Box_ UI cannot be activated using keyboard

### ISSUE THREAD

**Issue by huda866** [kyma-project/busola#3186]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High   
Affected Check Point:	ACC-270.6
Affected UI:	Combo Box
Short text:	ACC-270.6(Level A)_Kyma-dashboard_Combo Box_ UI cannot be activated using keyboard
1.2 Application Details
Application Name	Kyma-dashboard
Application URL:	https://canary.cockpit.btp.int.sap/cockpit#/globalaccount/7304343e-1ce0-412b-a955-5948bab9d2c8/subaccount/e0ec1134-2770-40a2-910b-2b8e32ec8f60/subaccountoverview
User:	SSO
Technology:	SAP UI5 
UI5 Version:	1.124.1
1.3 Test Environment Details:
GitHub	https://github.com/kyma-project/busola/issues
Web browser:	Google Chrome - Version 127.0.6533.73 (Official Build) (64-bit)
Operating System:	Windows 11 Enterprise Version 22H2
Screen Reader:	JAWS 2024.2405.83 ILM 64 bit for Windows
2. Screen Flow
•	Login into the application using the URL provided.
•	URL: https://canary.cockpit.btp.int.sap/cockpit#/globalaccount/7304343e-1ce0-412b-a955-5948bab9d2c8/subaccount/e0ec1134-2770-40a2-910b-2b8e32ec8f60/subaccountoverview.
•	Activate ‘Console URL:Link to dashboard’ in overview page.
•	Activate ‘Namespaces overview’ left navigation section.
•	Activate ‘Deployments’ item.
•	Activate ‘Create’ button.
3. Issue 
Synopsis: UI cannot be activated using keyboard.
Observed Behavior: 
Last list item in all unit of measurement combo box within ‘Deployments’ page is not keyboard accessible in forward navigation.
Expected Behavior: 
Each & every UI, must be accessible using the keyboard.
Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence all the UIs must be accessible with keyboard.

________________________________________________________________
Kindly refer to attached error and issue processing document for more details.

Regards,
Huda

**Comment by huda866** [comment]
[ACC-270.6(Level A)_Kyma-dashboard_Combo Box_ UI cannot be activated using keyboard.docx](https://github.com/user-attachments/files/16668265/ACC-270.6.Level.A._Kyma-dashboard_Combo.Box_.UI.cannot.be.activated.using.keyboard.docx)
[Issue processing-09.2021 (1).docx](https://github.com/user-attachments/files/16668266/Issue.processing-09.2021.1.docx)
